### ISSUE TITLE
[Bug Report] focus is unexpectedly given to the el-tooltip

### ISSUE THREAD

**Issue by FranckFreiburger** [ElemeFE/element#11838]
### Element UI version
2.4.3

### OS/Browsers version
any

### Vue version
2.5.16

### Reproduction Link
[https://jsfiddle.net/fs4zcj7d/](https://jsfiddle.net/fs4zcj7d/)

### Steps to reproduce
1/ click into field1
2/ press TAB key

### What is Expected?
focus to go from field to field

### What is actually happening?
focus is unexpectedly given to the el-tooltip

**Comment by ziyoung** [comment]
@FranckFreiburger the [tabindex](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex) of `el-form-item` is 0 so it can be focused. When `el-form-item` focus, the tooltip shows.
![image](https://user-images.githubusercontent.com/12947422/42314500-bfb8553c-8077-11e8-89c6-747fd7675b47.png)

**Comment by ziyoung** [comment]
@FranckFreiburger Maybe we should allow users to set tabindex. #11875

**Comment by FranckFreiburger** [comment]
Please, look at this issue that has been fixed https://github.com/ElemeFE/element/issues/8759
Before #8759 fix, tooltip was able to catch the focus, that was considered as an incorrect behavior.
Here we have the same issue but with a different element structure.
`<el-tooltip>` is not an input element, then it should never get the focus.

**Comment by danilobreda** [comment]
This bug is occurring on my project, why this is tagged as resolved?

**Comment by benmo1602** [comment]
![image](https://user-images.githubusercontent.com/22746527/79734578-8797ed00-8329-11ea-86b1-f24f3cf42d53.png)
slider  的 disabled 为true 时，   el-tooltip 的 tabindex = 0 ， 导致 按下tab 依然会获取焦点, 这不合理 ， 我没办法控制

**Comment by eloise-mcintyre** [comment]
Just tested this on version 2.8.2 and it still doesn't work
